sieve_of _eratosthenes/SieveOfEratosthenes.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n){
+	bool numbers[n+1];
+	int i,sqrt_n = sqrt(n)+1,j;
+	
+	for (i = 0; i < n+1; i++){
+		numbers[i] = true;      // assigning every number "true" 
+	}
+
+	for (i = 2; i < sqrt_n; i++){
+		if (numbers[i] == true){
+
+			for (j = 2*i; j < n+1; j=j+i){
```

Comment:
Please follow a consistent style. You seem to be preferring space before and after assignment/logical operators.  Eg `j = 2*i`. So make it `j = j+i`.